tests/blockchain/eth/contracts/base/test_issuer.py

```diff
@@ -20,31 +20,56 @@
 from eth_tester.exceptions import TransactionFailed
 from web3.contract import Contract
 
+from nucypher.blockchain.eth.token import NU
+
 SECRET_LENGTH = 32
 
 
 @pytest.fixture()
 def token(testerchain):
     # Create an ERC20 token
-    token, _ = testerchain.interface.deploy_contract('NuCypherToken', 2 * 10 ** 40)
+    token, _ = testerchain.interface.deploy_contract('NuCypherToken', _totalSupply=int(NU(2 * 10 ** 40, 'NuNit')))
     return token
 
 
 @pytest.mark.slow
 def test_issuer(testerchain, token):
+    mining_coefficient = 10 ** 43
```

Comment:
Can these be moved onto a subclass of `TokenEconomics`, perhaps `TestEconomics`, or similar?